src/OpenLoco/StationManager.cpp

```diff
@@ -133,7 +133,7 @@ namespace OpenLoco::StationManager
             {
                 // Zero unused station
                 station = {};
-                station.name = StringIds::empty;
+                station.name = StringIds::null;
```

Comment:
This entire line is probably not necessary anymore, given that the station struct has `StringIds::null` as the default value for the `name` member.

Reply:
Remove the line